### ISSUE TITLE
Multiline function arguments, last argument with no type or default

### ISSUE THREAD

**Issue by anlutro** [gerardroche/sublime-php-grammar#12]
In the following example, `$bar` will not receive the `variable.other.php` type, causing it not to be coloured. Adding a type (`MyType $bar`) or default value (`$bar = null`) fixes it.

```php
function f(
    $foo,
    $bar
) {}
```

**Comment by anlutro** [comment]
I was able to "fix" this by adding the following match to `function-arguments`:

```yml
    - match: '\$[a-zA-Z_\x{7f}-\x{ff}][a-zA-Z0-9_\x{7f}-\x{ff}]*'
      scope: variable.other.php
```

Do you think this is a viable solution? I have a gut feeling that modifying one of the existing ones to match arguments without type and defaults would be a better option.

**Comment by gerardroche** [comment]
The function argument match rules are not a straight forward as think they could be, but the smallest change could break a lot of stuff. I think it best to ensure good test coverage of the function argument syntax before making any changes.

**Comment by gerardroche** [comment]
![2f35f2b6-320b-11e5-938f-0ddfd6c46463](https://cloud.githubusercontent.com/assets/44148/8874836/92dd9b60-320c-11e5-9f07-0cd3cc70c565.png)

Re: https://github.com/atom/language-php/issues/91

**Comment by Ingramz** [comment]
I looked into this and the issue comes from `meta.function.argument.no-default.php` rule's last part: `(?=,|\\)|/[/*]|\\#)` - meaning that on the same line after the variable name, there has to be either a comma, closing parentheses or a beginning of a comment.

My initial solution was to add end of line marker `$` to the list of alternatives. Then I figured that that would make things such as
```php
<?php

function(
$a
$b
$c
) { }
```
legal. But that is already sort of broken by comments (works with current grammar):
```php
<?php

function(
$a # something
$b # comment
$c # etc
) { }
```

So unless someone wants to revamp the function arguments grammar, I think adding the end of line marker `$` to the list of alternatives would be an OK solution for that.
